test/error_messages.coffee

```diff
@@ -8,10 +8,10 @@ errCallback = (expectedErrorFormat) -> (err) ->
   err.colorful = no
   eq expectedErrorFormat, "#{err}"
   yes
-assertErrorFormat = (code, expectedErrorFormat) ->
+assertErrorFormatNoAst = (code, expectedErrorFormat) ->
```

Comment:
Adjusted so the "default" `assertErrorFormat()` includes AST checking, and renamed the existing `assertErrorFormat()` to `assertErrorFormatNoAst()` (for cases that don't also throw an error during AST generation)